applications: nrf_desktop: Add DT alias to configure QDEC in wheel (#14483)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

nRF Desktop
-----------

* Added the ``nrfdesktop-wheel-qdec`` DT alias support to :ref:`nrf_desktop_wheel`.

>   The alias must be used to specify QDEC instance used for scroll wheel if used board supports multiple QDEC instances (for example ``nrf54l15pdk_nrf54l15_cpuapp``).

Reviewer: ```suggestion
  You must use the alias to specify the QDEC instance used for scroll wheel, if your board supports multiple QDEC instances (for example ``nrf54l15pdk_nrf54l15_cpuapp``).
```

>   The alias does not need to be defined if used board supports only one QDEC instance, because in that case wheel module can rely on ``qdec`` DT label provided by the board.

Reviewer: ```suggestion
  You do not need to define the alias if your board supports only one QDEC instance, because in that case, the wheel module can rely on the ``qdec`` DT label provided by the board.
```

nRF Desktop
-----------

> Added:

Reviewer: ```suggestion
* Added:
```
We have these "headings" (Added, Updated, Fixed etc.) bulleted.
